content/03.methods.md

```diff
@@ -0,0 +1,32 @@
+## Methods
+
+### Datasets
+
+#### bioRxiv
+1. Describe how bioRxiv was obtained
+2. Describe metadata statistics on bioRxiv (number of preprints, number of preprints with multiple versions)
+
+#### PubMed Central
+1. Describe how PubMed central was obtained
+2. Describe metadata statistics on PubMed central (number of articles, how many articles were processed
+
+### Comparing Corpora

```

Comment:
Word counts, stopwords (if any), etc. The basics seem like they might be missing here. I am imagining that you will write them but maybe didn't note them yet :)